services/src/handlers/plots.rs

```diff
@@ -56,39 +61,60 @@ pub(crate) struct GetPlot {
 ///   "operator": {
 ///     "type": "Statistics",
 ///     "params": {},
-///     "rasterSources": [
-///       {
-///         "type": "MockRasterSource",
-///         "params": {
-///           "data": [
-///             {
-///               "time": {
-///                 "start": -8334632851200000,
-///                 "end": 8210298412799999
-///               },
-///               "tilePosition": [0, 0],
-///               "globalGeoTransform": {
-///                 "originCoordinate": { "x": 0.0, "y": 0.0 },
-///                 "xPixelSize": 1.0,
-///                 "yPixelSize": -1.0
-///               },
-///               "gridArray": {
-///                 "shape": {
-///                   "shapeArray": [3, 2]
+///     "sources": {
+///       "source": [
+///         {
+///           "type": "MockRasterSourcei32",
+///           "params": {
+///             "data": [
+///               {
+///                 "time": {
+///                   "start": -8334632851200000,
+///                   "end": 8210298412799999
+///                 },
+///                 "tilePosition": [0,0],
+///                 "globalGeoTransform": {
+///                   "originCoordinate": {"x": 0.0,"y": 0.0},
+///                   "xPixelSize": 1.0,
+///                   "yPixelSize": -1.0
+///                 },
+///                 "gridArray": {
+///                   "type": "grid",
+///                   "innerGrid": {
+///                     "shape": {
+///                       "shapeArray": [3, 2]
+///                     },
+///                     "data": [1, 2, 3, 4, 5, 6]
+///                   },
+///                   "validityMask": {
+///                     "shape": {
+///                       "shapeArray": [3,2]
+///                     },
+///                     "data": [true,true,true,true,true,true]
+///                   }
```

Comment:
Changing the type hasn't done anything. It still looks the same.

Reply:
Hm okay, but maybe we should have an easier example that is at least easier to do in Swagger.

Reply:
I intentionally created an example consisting of multiple steps because I wanted that the user understands what is happening. The only other way I know of to construct a vector source (inplace) is using MockFeatureCollectionSource. But I don't quite like it because it involves a large binary blob: [19f4b6c](https://github.com/geo-engine/geoengine/pull/667/commits/19f4b6c9ede78fabb7e1e870613c703595128e28)
Do you know of something better?

Reply:
[19f4b6c](https://github.com/geo-engine/geoengine/pull/667/commits/19f4b6c9ede78fabb7e1e870613c703595128e28)
[5567852](https://github.com/geo-engine/geoengine/pull/667/commits/5567852a1bbdcd1dd6f3be34e26348de10f35e62)